content/blog/2020/12/2020-12-03-election-results.adoc

```diff
@@ -0,0 +1,188 @@
+and we are excited to see contributors taking these roles.
+If you are interested to learn more, please see the blog post below.
+
+== Governance Board election details
+
+This year we had link:/blog/2020/10/28/election-candidates/#governance-board[nine candidates] participating in Jenkins Governance Board elections: 
+Andrey Falko, Ewelina Wilkosz, Frederic Gurr, Gavin Mogan, Justin Harringa,
+Mark Waite, Marky Jackson, Steven Terrana, and Zhao Xiaojie (Rick).
+All of them are awesome community leaders who actively contribute to the Jenkins project and
+represent its users.
+It would be an honor to have them on the Jenkins board.
+Regardless of the election results, we appreciate their participation and the time they invested in these elections.
+
+This year we were electing 2 governance board members.
+We were using the https://civs.cs.cornell.edu/[Condorcet Internet Voting Service] that allows voters to rank their choices rather than just picking their one favorite choice.
+You can find full voting results link:https://civs.cs.cornell.edu/cgi-bin/results.pl?id=E_f622a8a94f337478[here]:
+
+1. Mark Waite  (Condorcet winner: wins contests with all other choices)
+2. **Marky Jackson**  loses to Mark Waite by 48–12
+3. **Gavin Mogan**  loses to Mark Waite by 51–10, loses to Marky Jackson by 31–20
+4. Ewelina Wilkosz  loses to Mark Waite by 48–14, loses to Gavin Mogan by 29–28
+5. Justin Harringa  loses to Mark Waite by 51–11, loses to Ewelina Wilkosz by 35–16
+6. Steven Terrana  loses to Mark Waite by 47–16, loses to Justin Harringa by 20–19
+7. Zhao Xiaojie (Rick)  loses to Mark Waite by 57–5, loses to Steven Terrana by 25–24
+8. Frederic Gurr  loses to Mark Waite by 52–10, loses to Zhao Xiaojie (Rick) by 25–24
+9. Andrey Falko  loses to Mark Waite by 56–6, loses to Frederic Gurr by 26–13
+
+Although Mark Waite came first in the voting results,
+being on the board would violate the link:/project/board-election-process/#corporate-involvement[Corporate Involvement clause] which states that
+"the number of board members affiliated with one company must be less than 50%".
+Mark will continue to be link:/project/team-leads/#documentation[Documentation officer].
+Regardless of his official role, Mark has been leading many initiatives and helping a lot with various aspects of the community governance.
+
+Congratulations to Marky and Gavin, and thanks to all candidates!
+All new board members are elected for a 2-year term unless they decide to step down earlier.
+The estimated end of the term for them is December 02, 2022.
+We would also like to thank link:https://github.com/slide[Alex Earl] and link:https://github.com/rtyler[R. Tyler Croy] who step down from the Jenkins Governance Board this year.
+Thanks to them for all contributions and for continued community leadership.
+
+Statement from Marky Jackson:
+
+TIP: It is a tremendous honor to be elected to this roles. I am so humbled.
```

Comment:
Weird formatting choice for the statements.

Reply:
It is the best I found among available options. Happy to accept any sugestions

![image](https://user-images.githubusercontent.com/3000480/101049139-89928180-3583-11eb-87ec-4db6b58e408f.png)

Reply:
How about, you know, quote blocks?

> <img width="1557" alt="Screenshot 2020-12-03 at 17 36 06" src="https://user-images.githubusercontent.com/1831569/101059012-16423d00-358e-11eb-80fe-f4bab2d6ab3e.png">

(Accidentally left in a dash at the end of the quote)

Sample code:

```
[quote, "Marky Jackson, Jenkins Governance Board Member and Events Officer"]
____
It is a tremendous honor to be elected to this roles. I am so humbled.
____